symbols/overlay29.yml

```diff
@@ -235,6 +377,65 @@ overlay29:
         
         Has a special check to return false for Labyrinth Cave B10F (the Gabite boss fight).
         
+        return: bool
+    - name: DefenderAbilityIsActive
+      address:
+        NA:
+          - 0x22F96CC
+          - 0x2301A0C
+          - 0x230A940
+          - 0x2311B94
+          - 0x2322D64
+          - 0x2328634
+          - 0x2329F14
+          - 0x232BDD0
+          - 0x232DE20
+          - 0x2332A0C
+        EU:
+          - 0x22FA08D
+          - 0x2302438
+          - 0x230B3B4
+          - 0x23125F4
+          - 0x23237CC
+          - 0x23290A0
+          - 0x232A980
+          - 0x232C840
+          - 0x232E860
+          - 0x233344C
+      description: |-
+        Checks if a defender has an active ability that isn't disabled by an attacker's Mold Breaker.
```

Comment:
I just realized there are two variations of this function with different assembly. I think they're clearly the same function, but maybe the compiler optimized them differently. Can we add a line to the description mentioning this? Maybe something like:

```
Checks if a defender has an active ability that isn't disabled by an attacker's Mold Breaker.

This function has two variants, each with slightly different assembly. This is probably due to differences in compiler optimizations at different addresses.
```